libdrgn/python/util.c

```diff
@@ -78,6 +78,20 @@ int index_converter(PyObject *o, void *p)
 	}
 }
 
+int u64_converter(PyObject *o, void *p)
+{
+	uint64_t *arg = p;
+
+	if (o == Py_None)
+		return 1;
+
```

Comment:
This leaves the argument uninitialized if it is `None`, which I don't think is what you intended. If you drop this`, `PyNumber_Index()` will take care of the error checking.
```suggestion
```

Reply:
Ah, thank you for that catch. I switched up the cases when I was re-reading the function quickly.